src/style.scss

```diff
@@ -27,9 +27,24 @@
 //   ...
 // }
 
-.testClass {
-  color:  var(--auro-color-border-error-on-light);
-  border: 1px solid var(--auro-color-border-error-on-light);
-  display: inline-block;
-  padding: var(--auro-text-body-size-default);
+.badge {
+  align-items: center;
+  background-color: var(--auro-color-ui-default-on-light);
+  border-radius: 22px;
```

Comment:
This should be a percentage or a REM. Hard pix values are not consistent with any potential of dynamic sizing by the end user.

Reply:
This will be difficult for MOW and AScom if we choose the REM approach because we would not be able to override those styles like we can with fonts, would it be possible to keep this a constant?

Reply:
We can add the `fixed` feature. The default should be REM.

Reply:
And ... this should be a design token.

Reply:
I am on all of this.